source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/login/auth0-lock-auto-mount.tsx

```diff
@@ -8,9 +9,14 @@ export interface IProps {
 
 export default class Lock extends React.Component<IProps> {
   state = { loggedIn : false };
+  lockContainerId: string;
+
+  componentWillMount() {
+    this.lockContainerId = uuid();
+  }
 
   componentDidMount() {
-    const lock = getAuth0LockInstance();
+    const lock = getAuth0LockInstance({ container: this.lockContainerId});
```

Comment:
needed to tweak the login component a bit, because it was covering the `Contact Us` link.